_data/simple-icons.json

```diff
@@ -13204,6 +13204,12 @@
             "hex": "E9573F",
             "source": "https://rubygems.org/pages/about"
         },
+        {
+            "title": "RubyMine",
+            "hex": "000000",
+            "source": "https://www.jetbrains.com/company/brand/logos/",
+            "guidelines": "https://www.jetbrains.com/company/brand/"
```

Comment:
```suggestion
            "source": "https://www.jetbrains.com/company/brand/logos",
            "guidelines": "https://www.jetbrains.com/company/brand"
```

Reply:
Done in 5cc2150faff1c411341d691535fba548ccfb7b0c

For consistency. Though being nitty 🧐 canonical URLs are the ones with trailing slash

The guidelines `/brand/` one has trailing slash as canonical URL
```html
<link rel="canonical" href="https://www.jetbrains.com/company/brand/">
```
And if accessing the non-trailing slash, you get a redirect to the canonical one:

![image](https://github.com/simple-icons/simple-icons/assets/8050648/37c77885-d05b-443e-9303-026d3a8c71ba)

Same happens with the source `brand/logos` URL:

![image](https://github.com/simple-icons/simple-icons/assets/8050648/17b466ee-c037-4046-88cf-853dd7657762)

`/brand/logos/` then redirects to `/brand/#logos-and-icons-jetbrains-logos`

![image](https://github.com/simple-icons/simple-icons/assets/8050648/493ce3d0-b67f-4aa8-916d-7aac20057093)

If have some time will open a PR normalizing all JetBrains URLs to save some GET requests around

Reply:
We try to keep all urls without the trailing slash - it saves a few bytes per icon, and as our collection grows, every little helps!

Reply:
Sure, let's save some bytes then 👍 Not a big deal anyway as long the link works